packages/backend/src/api/request.ts

```diff
@@ -1,6 +1,5 @@
 import { ClerkAPIResponseError } from '@clerk/shared/error';
 import type { ClerkAPIError, ClerkAPIErrorJSON } from '@clerk/types';
-import deepmerge from 'deepmerge';
```

Comment:
I think you can remove this dependency from the `package.json` now since this was the only place where it was used

Reply:
I will drop this as part of the `sdk-782-drop-deprecations-backend` since the usage was removed from there.

Reply:
https://github.com/clerk/javascript/pull/1899/commits/286843b8aa907588f45910c1e3a82ad79dedbb0e